fitbenchmarking/controllers/minuit_controller.py

```diff
@@ -24,6 +25,11 @@ def __init__(self, cost_func):
                 :class:`~fitbenchmarking.cost_func.base_cost_func.CostFunc`
 
         """
+
+        if int(iminuit.__version__[:1]) < 2:
+            raise Exception("iminuit version " + iminuit.__version__ + 
```

Comment:
Where possible I'd advise against using the raw `Exception`, we have some more specific ones in `utils/exceptions.py` and if none of those fit we should make a new one.